controller/collector/spec.go

```diff
@@ -0,0 +1 @@
+package collector
```

Comment:
We can delete the file if it does not serve any purpose. Usually we define interfaces here but there does not seem to be any. So dropping the spec file should be fine.